From 98987c65df7b3f8a5cf818e7d32533814e301c25 Mon Sep 17 00:00:00 2001
From: Daniel Landau <daniel@landau.fi>
Date: Sat, 11 Mar 2017 22:53:18 +0200
Subject: [PATCH 1/2] Revert "Enable Nitrous BT power management driver for
 both Sparrow and Wren."

This reverts commit fd660c7daa1356bed24d26f97b667e76d64d7689.
---
 arch/arm/boot/dts/qcom/sparrow_er_105_msm8226.dtsi | 10 ----
 arch/arm/boot/dts/qcom/sparrow_er_msm8226.dtsi     | 10 ----
 arch/arm/boot/dts/qcom/sparrow_evb_msm8226.dtsi    | 11 ----
 .../arm/boot/dts/qcom/sparrow_pr3_105_msm8226.dtsi | 10 ----
 arch/arm/boot/dts/qcom/sparrow_pr3_msm8226.dtsi    | 10 ----
 arch/arm/boot/dts/qcom/sparrow_pr_105_msm8226.dtsi | 10 ----
 arch/arm/boot/dts/qcom/sparrow_pr_msm8226.dtsi     | 10 ----
 arch/arm/boot/dts/qcom/sparrow_sr2_msm8226.dtsi    | 10 ----
 arch/arm/boot/dts/qcom/sparrow_sr_pni_msm8226.dtsi | 10 ----
 arch/arm/boot/dts/qcom/sparrow_sr_ql_msm8226.dtsi  | 10 ----
 arch/arm/boot/dts/qcom/wren_er_105_msm8226.dtsi    | 10 ----
 arch/arm/boot/dts/qcom/wren_er_msm8226.dtsi        | 10 ----
 arch/arm/boot/dts/qcom/wren_evb_sr_msm8226.dtsi    | 10 ----
 arch/arm/boot/dts/qcom/wren_pr3_105_msm8226.dtsi   | 10 ----
 arch/arm/boot/dts/qcom/wren_pr3_msm8226.dtsi       | 10 ----
 arch/arm/boot/dts/qcom/wren_pr_105_msm8226.dtsi    | 10 ----
 arch/arm/boot/dts/qcom/wren_pr_msm8226.dtsi        | 10 ----
 arch/arm/mach-msm/board-8226.c                     | 61 ++++++++++++++++++++++
 arch/arm/mach-msm/sparrow/Makefile                 |  2 +-
 arch/arm/mach-msm/wren/Makefile                    |  2 +-
 drivers/BCM_LPM/Makefile                           |  2 +-
 21 files changed, 64 insertions(+), 174 deletions(-)

diff --git a/arch/arm/boot/dts/qcom/sparrow_er_105_msm8226.dtsi b/arch/arm/boot/dts/qcom/sparrow_er_105_msm8226.dtsi
index 660918622417..1dd4bc7e7f39 100755
--- a/arch/arm/boot/dts/qcom/sparrow_er_105_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/sparrow_er_105_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/sparrow_er_msm8226.dtsi b/arch/arm/boot/dts/qcom/sparrow_er_msm8226.dtsi
index 132b1a7f5392..79d2009b6185 100755
--- a/arch/arm/boot/dts/qcom/sparrow_er_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/sparrow_er_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/sparrow_evb_msm8226.dtsi b/arch/arm/boot/dts/qcom/sparrow_evb_msm8226.dtsi
index 6af5b428a9cc..64d1a648da9b 100755
--- a/arch/arm/boot/dts/qcom/sparrow_evb_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/sparrow_evb_msm8226.dtsi
@@ -278,17 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		reset-gpio = <&msmgpio 36 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/sparrow_pr3_105_msm8226.dtsi b/arch/arm/boot/dts/qcom/sparrow_pr3_105_msm8226.dtsi
index 4ae9452c49b7..2d8ac2896ad1 100755
--- a/arch/arm/boot/dts/qcom/sparrow_pr3_105_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/sparrow_pr3_105_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/sparrow_pr3_msm8226.dtsi b/arch/arm/boot/dts/qcom/sparrow_pr3_msm8226.dtsi
index dba8907d52ab..c18d5e609b43 100755
--- a/arch/arm/boot/dts/qcom/sparrow_pr3_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/sparrow_pr3_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/sparrow_pr_105_msm8226.dtsi b/arch/arm/boot/dts/qcom/sparrow_pr_105_msm8226.dtsi
index f2b540293d32..613e2bc73a63 100755
--- a/arch/arm/boot/dts/qcom/sparrow_pr_105_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/sparrow_pr_105_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/sparrow_pr_msm8226.dtsi b/arch/arm/boot/dts/qcom/sparrow_pr_msm8226.dtsi
index 49733411e1ef..6308c5d5f5e1 100755
--- a/arch/arm/boot/dts/qcom/sparrow_pr_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/sparrow_pr_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/sparrow_sr2_msm8226.dtsi b/arch/arm/boot/dts/qcom/sparrow_sr2_msm8226.dtsi
index 64fcb8b3aaff..3df0e4731033 100755
--- a/arch/arm/boot/dts/qcom/sparrow_sr2_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/sparrow_sr2_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/sparrow_sr_pni_msm8226.dtsi b/arch/arm/boot/dts/qcom/sparrow_sr_pni_msm8226.dtsi
index eda06f6491a7..4db968c6b5ae 100755
--- a/arch/arm/boot/dts/qcom/sparrow_sr_pni_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/sparrow_sr_pni_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/sparrow_sr_ql_msm8226.dtsi b/arch/arm/boot/dts/qcom/sparrow_sr_ql_msm8226.dtsi
index 68d0cdb18216..1b8b99892f41 100755
--- a/arch/arm/boot/dts/qcom/sparrow_sr_ql_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/sparrow_sr_ql_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/wren_er_105_msm8226.dtsi b/arch/arm/boot/dts/qcom/wren_er_105_msm8226.dtsi
index 84807b1c6869..5c0c7926f722 100755
--- a/arch/arm/boot/dts/qcom/wren_er_105_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/wren_er_105_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/wren_er_msm8226.dtsi b/arch/arm/boot/dts/qcom/wren_er_msm8226.dtsi
index 638a1749f712..6b09b24fcd0b 100755
--- a/arch/arm/boot/dts/qcom/wren_er_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/wren_er_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/wren_evb_sr_msm8226.dtsi b/arch/arm/boot/dts/qcom/wren_evb_sr_msm8226.dtsi
index a10fbb131ef7..6a9f10b0b155 100755
--- a/arch/arm/boot/dts/qcom/wren_evb_sr_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/wren_evb_sr_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/wren_pr3_105_msm8226.dtsi b/arch/arm/boot/dts/qcom/wren_pr3_105_msm8226.dtsi
index 0f3b8e67452e..82aaf71117bd 100755
--- a/arch/arm/boot/dts/qcom/wren_pr3_105_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/wren_pr3_105_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/wren_pr3_msm8226.dtsi b/arch/arm/boot/dts/qcom/wren_pr3_msm8226.dtsi
index d2f1d4d21127..12a61df72647 100755
--- a/arch/arm/boot/dts/qcom/wren_pr3_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/wren_pr3_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/wren_pr_105_msm8226.dtsi b/arch/arm/boot/dts/qcom/wren_pr_105_msm8226.dtsi
index e24fdeaecef8..a2cd1ee1421a 100755
--- a/arch/arm/boot/dts/qcom/wren_pr_105_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/wren_pr_105_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/boot/dts/qcom/wren_pr_msm8226.dtsi b/arch/arm/boot/dts/qcom/wren_pr_msm8226.dtsi
index de8306a3dd85..7d4cced17e61 100755
--- a/arch/arm/boot/dts/qcom/wren_pr_msm8226.dtsi
+++ b/arch/arm/boot/dts/qcom/wren_pr_msm8226.dtsi
@@ -278,16 +278,6 @@
 				<86 512 0 0>,
 				<86 512 500 800>;
 	};
-
-	bt_nitrous {
-		compatible = "goog,nitrous";
-		uart-port = <0>;
-		power-gpio = <&msmgpio 34 0>;
-		host-wake-gpio = <&msmgpio 48 0>;
-		host-wake-polarity = <1>;
-		dev-wake-gpio = <&msmgpio 61 0>;
-		dev-wake-polarity = <1>;
-	};
 	// ASUS_BSP BerylHou ---
 
 	uart_2:serial@f991f000 { //ASUS_BSP BerylHou +++
diff --git a/arch/arm/mach-msm/board-8226.c b/arch/arm/mach-msm/board-8226.c
index 07a1197637d9..a238056ad9c4 100755
--- a/arch/arm/mach-msm/board-8226.c
+++ b/arch/arm/mach-msm/board-8226.c
@@ -76,6 +76,37 @@ void __init device_gpiomux_init(void)
 }
 //--ASUS_BSP : add for gpio
 
+//ASUS_BSP BerylHou +++ "Add for BT porting"
+static struct resource bluesleep_resources[] = {
+	{
+		.name	= "gpio_host_wake", //gpio bt_wake_up_host
+		.start	= -1,
+		.end	= -1, 
+		.flags	= IORESOURCE_IO,
+	},
+	{
+		.name	= "gpio_ext_wake", //gpio host_wake_up_bt
+		.start	= -1, 
+		.end	= -1, 
+		.flags	= IORESOURCE_IO,
+	},
+	{
+		.name	= "host_wake", //IRQ bt wake up host
+		.start	= -1, 
+		.end	= -1, 
+		.flags	= IORESOURCE_IRQ,
+	},
+};
+
+static struct platform_device msm_bluesleep_device = {
+	.name		= "bluesleep",
+	.id		= -1,
+	.num_resources	= ARRAY_SIZE(bluesleep_resources),
+	.resource	= bluesleep_resources,
+};
+
+//ASUS_BSP BerylHou ---
+
 static struct of_dev_auxdata msm8226_auxdata_lookup[] __initdata = {
 	OF_DEV_AUXDATA("qcom,sdhci-msm", 0xF9824900, \
 			"msm_sdcc.1", NULL),
@@ -90,6 +121,35 @@ static struct of_dev_auxdata msm8226_auxdata_lookup[] __initdata = {
 	{}
 };
 
+//ASUS_BSP BerylHou +++ "Add for BT porting"
+static void gpio_bt_init(void)
+{
+	printk("[bt] gpio init");
+	
+	bluesleep_resources[0].start = GPIO_BT_WAKE_UP_HOST;
+	bluesleep_resources[0].end = GPIO_BT_WAKE_UP_HOST;
+
+	if (g_ASUS_hwID == SPARROW_EVB) {
+		bluesleep_resources[1].start = GPIO_HOST_WAKE_UP_BT_EVB2;
+		bluesleep_resources[1].end = GPIO_HOST_WAKE_UP_BT_EVB2;
+	} else {
+		bluesleep_resources[1].start = GPIO_HOST_WAKE_UP_BT_SR;
+		bluesleep_resources[1].end = GPIO_HOST_WAKE_UP_BT_SR;
+	}
+
+	bluesleep_resources[2].start = gpio_to_irq(GPIO_BT_WAKE_UP_HOST);
+	bluesleep_resources[2].end = gpio_to_irq(GPIO_BT_WAKE_UP_HOST);
+
+}
+
+static void __init board_8226_bluesleep_setup(void)
+{
+	printk("set up bt sleep mode\n");
+	gpio_bt_init();
+	platform_device_register(&msm_bluesleep_device);
+}
+//ASUS_BSP BerylHou ---
+
 static void __init msm8226_reserve(void)
 {
 	of_scan_flat_dt(dt_scan_for_memory_reserve, NULL);
@@ -167,6 +227,7 @@ void __init msm8226_init(void)
 	device_gpiomux_init();
 //---ASUS_BSP : add for gpio
 	msm8226_add_drivers();
+	board_8226_bluesleep_setup(); //ASUS_BSP BerylHou +++ "BT porting"
 }
 
 static const char *msm8226_dt_match[] __initconst = {
diff --git a/arch/arm/mach-msm/sparrow/Makefile b/arch/arm/mach-msm/sparrow/Makefile
index 50e20161198b..188c43e5c022 100644
--- a/arch/arm/mach-msm/sparrow/Makefile
+++ b/arch/arm/mach-msm/sparrow/Makefile
@@ -1,3 +1,3 @@
 # Add the ASUS customized modules into kernel
 
-obj-y += gpio_pinmux.o
+obj-y += gpio_pinmux.o rfkill.o
diff --git a/arch/arm/mach-msm/wren/Makefile b/arch/arm/mach-msm/wren/Makefile
index 50e20161198b..188c43e5c022 100644
--- a/arch/arm/mach-msm/wren/Makefile
+++ b/arch/arm/mach-msm/wren/Makefile
@@ -1,3 +1,3 @@
 # Add the ASUS customized modules into kernel
 
-obj-y += gpio_pinmux.o
+obj-y += gpio_pinmux.o rfkill.o
diff --git a/drivers/BCM_LPM/Makefile b/drivers/BCM_LPM/Makefile
index 72013a6043d4..c13de5ffea01 100644
--- a/drivers/BCM_LPM/Makefile
+++ b/drivers/BCM_LPM/Makefile
@@ -1 +1 @@
-obj-$(CONFIG_BCM_BT_LPM) += nitrous.o
+obj-$(CONFIG_BCM_BT_LPM) += bluesleep.o
-- 
2.12.0

